src/components/pages/Home/index.tsx

```diff
@@ -1,388 +1,105 @@
-  },
-  {
-    key: 'mint',
-    children: (
-      <>
-        <SymbolBlock>
-          <Symbol>
-            <div>Mint</div>
-            <i>+</i>
-          </Symbol>
-        </SymbolBlock>
-        <Block>
-          <P>
-            Get mUSD by depositing your USDC, DAI, TUSD or USDT at a 1:1 ratio.
-          </P>
-          <P>
-            <Link to="/mint">Go to mint</Link>
-          </P>
-        </Block>
-      </>
-    ),
-  },
-  {
-    key: 'save',
-    children: (
-      <>
-        <SymbolBlock>
-          <Symbol>
-            <div>Save</div>
-            <i>×</i>
-          </Symbol>
-        </SymbolBlock>
-        <Block>
-          <P>Earn mUSD&rsquo;s native interest rate.</P>
-          <P>
-            <Link to="/save">Go to save</Link>
-          </P>
-        </Block>
-      </>
-    ),
-  },
-  {
-    key: 'swap',
-    children: (
-      <>
-        <SymbolBlock>
-          <Symbol>
-            <div>Swap</div>
-            <i>=</i>
-          </Symbol>
-        </SymbolBlock>
-        <Block>
-          <P>
-            Swap between stablecoins at zero slippage (a trading fee applies).
-          </P>
-          <P>
-            <Link to="/swap">Go to swap</Link>
-          </P>
-        </Block>
-      </>
-    ),
-  },
-  {
-    key: 'earn',
-    children: (
-      <>
-        <SymbolBlock>
-          <Symbol>
-            <div>Earn</div>
-            <i>$</i>
-          </Symbol>
-        </SymbolBlock>
-        <Block>
-          <P>
-            mStable rewards users who contribute mUSD liquidity across the DeFi
-            ecosystem.
-          </P>
-          <P>
-            <Link to="/earn">Go to earn</Link>
-          </P>
-        </Block>
-      </>
-    ),
-  },
-  {
-    key: 'govern',
-    children: (
-      <>
-        <SymbolBlock>
-          <Symbol>
-            <div>govern</div>
-            <i>
-              <GovernanceIcon />
-            </i>
-          </Symbol>
-        </SymbolBlock>
-        <Block>
-          <P>Participate in system goverance</P>
-          <P>
-            <a href="https://governance.mstable.org">Go to governance</a>
-          </P>
-        </Block>
-      </>
-    ),
-  },
-  {
-    key: 'get-started',
-    children: <GetStarted />,
-  },
-];
+const Mbtc = styled(Musd)`
+  background: ${({ theme }) => theme.color.gold};
+  border: 10px solid #ffedd1;
+`;
 
 export const Home: FC = () => {
-  const connect = useConnect();
-  const [activeIdx, setActiveIdx] = useState<number>(0);
-
-  const next = useCallback(() => {
-    if (activeIdx < HOME_STEPS.length - 1) {
-      setActiveIdx(activeIdx + 1);
-    } else {
-      connect();
-    }
-  }, [setActiveIdx, activeIdx, connect]);
-
-  const previous = useCallback(() => {
-    if (activeIdx > 0) {
-      setActiveIdx(activeIdx - 1);
-    }
-  }, [setActiveIdx, activeIdx]);
-
+  const selectMasset = useSetSelectedMasset();
+  const history = useHistory();
+  const handleMusdClick = useCallback(() => {
```

Comment:
This is duplicated elsewhere, and with the mBTC version - maybe it might be better to export a hook that returns a callback like `(massetName: MassetName) => void` (that selects the masset and changes the route?)